vasern/src/plugins/vasern-objectid/ObjectID.js

```diff
@@ -21,7 +21,7 @@ export default class ObjectID {
   // machine id
   mid: string;
 
-  constructor(prop: { mid: ?string }) {
+  constructor(prop: { mid: ?string }) { // Question: Is the constructor necessary? It's not used anywhere and always generates a warning
```

Comment:
It was originally designed for distributed database (so each app will have a specific `mid`/machine id) to prevent ID collision.

Good point though, I'll remove the warning by one of the following method:
1. Remove the constructor and add a `setMID` method
2. Set `prop` as optiontal then just generate an `mid` if there is no `prop`

Reply:
I'd prefer the first option as it is less error prone. Letting developers set the mid might lead to more collision than we actually wanted to prevent in the end (imagine copy-pasted code where the mid isn't changed appropriately.

Reply:
Oh alright, sorry I didn't make it clear enough. You are right, the `mid` isn't support to be set by developers. It might be requested from the server or auto-generated, then saved in the local DB. Every time the app started, it will load the `mid` and use the `setMID` method.

I do think the first option would be more appropriate. I'll take note and make an update